2023-04-19 09:42:22 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get state
2023-04-19 09:42:22 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to set attempt sync config
2023-04-19 09:42:22 [32mINFO[m i.a.c.t.s.DefaultTaskQueueMapper(getTaskQueue):31 - Called DefaultTaskQueueMapper getTaskQueue for geography auto
2023-04-19 09:42:22 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/2/0/logs.log
2023-04-19 09:42:22 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.44.0
2023-04-19 09:42:22 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to save workflow id for cancellation
2023-04-19 09:42:22 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get the source for heartbeat
2023-04-19 09:42:22 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get the source definition
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):93 - Setting up source...
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):97 - Setting up destination...
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable METRIC_CLIENT: ''
2023-04-19 09:42:22 [33mWARN[m i.a.m.l.MetricClientFactory(initialize):60 - Metric client is already initialized to 
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):106 - Setting up replication worker...
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):161 - start sync worker. job id: 2 attempt id: 0
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):163 - Committing states from replication activity
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):166 - Committing stats from replication activity
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):182 - configured sync modes: {null.my_stream=full_refresh - overwrite}
2023-04-19 09:42:22 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:22 [32mINFO[m i.a.w.i.DefaultAirbyteDestination(start):92 - Running destination...
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:22 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- START REPLICATION -----
2023-04-19 09:42:22 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:22 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/destination-mysql:0.1.20 exists...
2023-04-19 09:42:22 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/destination-mysql:0.1.20 was found locally.
2023-04-19 09:42:22 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = destination-mysql-write-2-0-qwqox with resources io.airbyte.config.ResourceRequirements@448e6ca4[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:22 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/0 --log-driver none --name destination-mysql-write-2-0-qwqox --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/destination-mysql:0.1.20 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=0 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/destination-mysql:0.1.20 write --config destination_config.json --catalog destination_catalog.json
2023-04-19 09:42:22 [32mINFO[m i.a.w.i.VersionedAirbyteMessageBufferedWriterFactory(createWriter):41 - Writing messages to protocol version 0.2.0
2023-04-19 09:42:22 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:22 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:22 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/source-customize-api:0.0.1 exists...
2023-04-19 09:42:22 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/source-customize-api:0.0.1 was found locally.
2023-04-19 09:42:22 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = source-customize-api-read-2-0-ccmbp with resources io.airbyte.config.ResourceRequirements@4aff6d0f[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:22 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/0 --log-driver none --name source-customize-api-read-2-0-ccmbp --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/source-customize-api:0.0.1 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=0 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/source-customize-api:0.0.1 read --config source_config.json --catalog source_catalog.json
2023-04-19 09:42:22 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromDstRunnable$4):310 - Destination output thread started.
2023-04-19 09:42:22 [32mINFO[m i.a.w.i.HeartbeatTimeoutChaperone(runWithHeartbeatThread):94 - Starting source heartbeat check. Will check every 1 minutes.
2023-04-19 09:42:22 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$8):376 - Replication thread started.
2023-04-19 09:42:23 [43mdestination[0m > SLF4J: Class path contains multiple SLF4J bindings.
2023-04-19 09:42:23 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:23 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:23 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:23 [43mdestination[0m > SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
2023-04-19 09:42:23 [43mdestination[0m > SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
2023-04-19 09:42:23 [44msource[0m > Starting syncing SourceCustomizeAPI
2023-04-19 09:42:23 [44msource[0m > {"status":500,"message":"用户信息不能为空","data":null,"error":null}
2023-04-19 09:42:23 [44msource[0m > 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 74, in handle_http_error
    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]
KeyError: 401

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/airbyte/integration_code/main.py", line 13, in <module>
    launch(source, sys.argv[1:])
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 131, in launch
    for message in source_entrypoint.run(parsed_args):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 122, in run
    for message in generator:
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py", line 110, in read
    stream_is_available, error = stream_instance.check_availability(logger, self)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py", line 190, in check_availability
    return self.availability_strategy.check_availability(self, logger, source)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 51, in check_availability
    return self.handle_http_error(stream, logger, source, error)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 81, in handle_http_error
    raise error
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 45, in check_availability
    get_first_record_for_slice(stream, stream_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py", line 38, in get_first_record_for_slice
    return next(records_for_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 423, in read_records
    yield from self._read_pages(
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 439, in _read_pages
    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 462, in _fetch_next_page
    response = self._send_request(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 364, in _send_request
    return backoff_handler(user_backoff_handler)(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 331, in _send
    raise exc
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 328, in _send
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
2023-04-19 09:42:24 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$8):447 - Total records read: 1 (0 bytes)
2023-04-19 09:42:24 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$8):456 - Schema validation was performed to a max of 10 records with errors per stream.
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):136 - starting destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationCliParser(parseOptions):118 - integration args: {catalog=destination_catalog.json, write=null, config=destination_config.json}"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):123 - Running integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):124 - Command: WRITE"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):125 - Integration config: IntegrationConfig{command=WRITE, configPath='destination_config.json', catalogPath='destination_catalog.json', statePath='null'}"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword order - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword examples - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword multiline - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.b.s.SshTunnel(getInstance):170 - Starting connection with method: NO_TUNNEL"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):80 - HikariPool-1 - Starting..."
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):82 - HikariPool-1 - Start completed."
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$toWriteConfig$0):98 - Write config: WriteConfig{streamName=my_stream, namespace=null, outputSchemaName=api, tmpTableName=_airbyte_tmp_pan_my_stream, outputTableName=_airbyte_raw_my_stream, syncMode=overwrite}"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.d.b.BufferedStreamConsumer(startTracked):116 - class io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer started."
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):123 - Preparing tmp tables in destination started for 1 streams"
2023-04-19 09:42:24 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:24 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):127 - Preparing tmp table in destination started for stream my_stream. schema: api, tmp table name: _airbyte_tmp_pan_my_stream"
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):133 - Preparing tables in destination completed."
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.b.FailureTrackingAirbyteMessageConsumer(close):65 - Airbyte message consumer: succeeded."
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.b.BufferedStreamConsumer(close):171 - executing on success close procedure."
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.SqlOperations(onDestinationCloseOperations):138 - No onDestinationCloseOperations required for this destination."
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):163 - Finalizing tables in destination started for 1 streams"
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):168 - Finalizing stream my_stream. schema api, tmp table _airbyte_tmp_pan_my_stream, final table _airbyte_raw_my_stream"
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):181 - Executing finalization of tables."
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):183 - Finalizing tables in destination completed."
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):186 - Cleaning tmp tables in destination started for 1 streams"
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):190 - Cleaning tmp table in destination started for stream my_stream. schema api, tmp table name: _airbyte_tmp_pan_my_stream"
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):195 - Cleaning tmp tables in destination completed."
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):171 - Completed integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:25 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):138 - completed destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:25 [1;31mERROR[m i.a.w.g.DefaultReplicationWorker(replicate):278 - Sync worker failed.
io.airbyte.workers.internal.exception.SourceException: Source process exited with non-zero exit code 1
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:475) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
	Suppressed: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.
		at io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.replicate(DefaultReplicationWorker.java:214) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.run(DefaultReplicationWorker.java:190) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.run(DefaultReplicationWorker.java:91) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$5(TemporalAttemptExecution.java:195) ~[io.airbyte-airbyte-workers-0.44.0.jar:?]
		at java.lang.Thread.run(Thread.java:1589) ~[?:?]
2023-04-19 09:42:25 [32mINFO[m i.a.w.g.DefaultReplicationWorker(getReplicationOutput):561 - sync summary: {
  "status" : "failed",
  "recordsSynced" : 0,
  "bytesSynced" : 0,
  "startTime" : 1681897342671,
  "endTime" : 1681897345335,
  "totalStats" : {
    "bytesCommitted" : 0,
    "bytesEmitted" : 0,
    "destinationStateMessagesEmitted" : 0,
    "destinationWriteEndTime" : 1681897345331,
    "destinationWriteStartTime" : 1681897342775,
    "meanSecondsBeforeSourceStateMessageEmitted" : 0,
    "maxSecondsBeforeSourceStateMessageEmitted" : 0,
    "maxSecondsBetweenStateMessageEmittedandCommitted" : 0,
    "meanSecondsBetweenStateMessageEmittedandCommitted" : 0,
    "recordsEmitted" : 0,
    "recordsCommitted" : 0,
    "replicationEndTime" : 1681897345334,
    "replicationStartTime" : 1681897342671,
    "sourceReadEndTime" : 1681897344133,
    "sourceReadStartTime" : 1681897342719,
    "sourceStateMessagesEmitted" : 0
  },
  "streamStats" : [ ]
}
2023-04-19 09:42:25 [32mINFO[m i.a.w.g.DefaultReplicationWorker(getReplicationOutput):562 - failures: [ {
  "failureOrigin" : "source",
  "failureType" : "system_error",
  "internalMessage" : "401 Client Error:  for url: http://192.168.100.158:9098/category/build/1",
  "externalMessage" : "Something went wrong in the connector. See the logs for more details.",
  "metadata" : {
    "attemptNumber" : 0,
    "jobId" : 2,
    "from_trace_message" : true,
    "connector_command" : "read"
  },
  "stacktrace" : "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 74, in handle_http_error\n    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]\nKeyError: 401\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/airbyte/integration_code/main.py\", line 13, in <module>\n    launch(source, sys.argv[1:])\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 131, in launch\n    for message in source_entrypoint.run(parsed_args):\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 122, in run\n    for message in generator:\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 110, in read\n    stream_is_available, error = stream_instance.check_availability(logger, self)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py\", line 190, in check_availability\n    return self.availability_strategy.check_availability(self, logger, source)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 51, in check_availability\n    return self.handle_http_error(stream, logger, source, error)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 81, in handle_http_error\n    raise error\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 45, in check_availability\n    get_first_record_for_slice(stream, stream_slice)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py\", line 38, in get_first_record_for_slice\n    return next(records_for_slice)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 423, in read_records\n    yield from self._read_pages(\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 439, in _read_pages\n    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 462, in _fetch_next_page\n    response = self._send_request(request, request_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 364, in _send_request\n    return backoff_handler(user_backoff_handler)(request, request_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/backoff/_sync.py\", line 105, in retry\n    ret = target(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/backoff/_sync.py\", line 105, in retry\n    ret = target(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 331, in _send\n    raise exc\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 328, in _send\n    response.raise_for_status()\n  File \"/usr/local/lib/python3.9/site-packages/requests/models.py\", line 1021, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1\n",
  "timestamp" : 1681897343973
}, {
  "failureOrigin" : "source",
  "internalMessage" : "Source process exited with non-zero exit code 1",
  "externalMessage" : "Something went wrong within the source connector",
  "metadata" : {
    "attemptNumber" : 0,
    "jobId" : 2,
    "connector_command" : "read"
  },
  "stacktrace" : "io.airbyte.workers.internal.exception.SourceException: Source process exited with non-zero exit code 1\n\tat io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:475)\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1589)\n",
  "timestamp" : 1681897344201
} ]
2023-04-19 09:42:25 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-04-19 09:42:25 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:25 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- END REPLICATION -----
2023-04-19 09:42:25 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:25 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$3):197 - sync summary: io.airbyte.config.StandardSyncOutput@2929d7e2[standardSyncSummary=io.airbyte.config.StandardSyncSummary@1a597f85[status=failed,recordsSynced=0,bytesSynced=0,startTime=1681897342671,endTime=1681897345335,totalStats=io.airbyte.config.SyncStats@6735f820[bytesCommitted=0,bytesEmitted=0,destinationStateMessagesEmitted=0,destinationWriteEndTime=1681897345331,destinationWriteStartTime=1681897342775,estimatedBytes=<null>,estimatedRecords=<null>,meanSecondsBeforeSourceStateMessageEmitted=0,maxSecondsBeforeSourceStateMessageEmitted=0,maxSecondsBetweenStateMessageEmittedandCommitted=0,meanSecondsBetweenStateMessageEmittedandCommitted=0,recordsEmitted=0,recordsCommitted=0,replicationEndTime=1681897345334,replicationStartTime=1681897342671,sourceReadEndTime=1681897344133,sourceReadStartTime=1681897342719,sourceStateMessagesEmitted=0,additionalProperties={}],streamStats=[],additionalProperties={}],normalizationSummary=<null>,webhookOperationSummary=<null>,state=<null>,outputCatalog=io.airbyte.protocol.models.ConfiguredAirbyteCatalog@5da5a2c4[streams=[io.airbyte.protocol.models.ConfiguredAirbyteStream@3ad4f855[stream=io.airbyte.protocol.models.AirbyteStream@6ba01ddd[name=my_stream,jsonSchema={"$schema":"http://json-schema.org/draft-07/schema","type":"object","required":["url"],"properties":{"url":{"description":"The URL for the API endpoint.","type":"string"},"token":{"description":"The API token for authentication.","type":"string"}}},supportedSyncModes=[full_refresh],sourceDefinedCursor=<null>,defaultCursorField=[],sourceDefinedPrimaryKey=[],namespace=<null>,additionalProperties={}],syncMode=full_refresh,cursorField=[],destinationSyncMode=overwrite,primaryKey=[],additionalProperties={}]],additionalProperties={}],failures=[io.airbyte.config.FailureReason@239c1c71[failureOrigin=source,failureType=system_error,internalMessage=401 Client Error:  for url: http://192.168.100.158:9098/category/build/1,externalMessage=Something went wrong in the connector. See the logs for more details.,metadata=io.airbyte.config.Metadata@535f3944[additionalProperties={attemptNumber=0, jobId=2, from_trace_message=true, connector_command=read}],stacktrace=Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 74, in handle_http_error
    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]
KeyError: 401

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/airbyte/integration_code/main.py", line 13, in <module>
    launch(source, sys.argv[1:])
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 131, in launch
    for message in source_entrypoint.run(parsed_args):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 122, in run
    for message in generator:
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py", line 110, in read
    stream_is_available, error = stream_instance.check_availability(logger, self)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py", line 190, in check_availability
    return self.availability_strategy.check_availability(self, logger, source)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 51, in check_availability
    return self.handle_http_error(stream, logger, source, error)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 81, in handle_http_error
    raise error
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 45, in check_availability
    get_first_record_for_slice(stream, stream_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py", line 38, in get_first_record_for_slice
    return next(records_for_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 423, in read_records
    yield from self._read_pages(
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 439, in _read_pages
    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 462, in _fetch_next_page
    response = self._send_request(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 364, in _send_request
    return backoff_handler(user_backoff_handler)(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 331, in _send
    raise exc
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 328, in _send
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
,retryable=<null>,timestamp=1681897343973,additionalProperties={}], io.airbyte.config.FailureReason@4e54652a[failureOrigin=source,failureType=<null>,internalMessage=Source process exited with non-zero exit code 1,externalMessage=Something went wrong within the source connector,metadata=io.airbyte.config.Metadata@2571c7c8[additionalProperties={attemptNumber=0, jobId=2, connector_command=read}],stacktrace=io.airbyte.workers.internal.exception.SourceException: Source process exited with non-zero exit code 1
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:475)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1589)
,retryable=<null>,timestamp=1681897344201,additionalProperties={}]],commitStateAsap=true,additionalProperties={}]
2023-04-19 09:42:25 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$3):202 - Sync summary length: 6593
2023-04-19 09:42:25 [32mINFO[m i.a.c.t.TemporalUtils(withBackgroundHeartbeat):307 - Stopping temporal heartbeating...
2023-04-19 09:42:28 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to Get a connection by connection Id
2023-04-19 09:42:28 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get the most recent source actor catalog
2023-04-19 09:42:29 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to Get a connection by connection Id
2023-04-19 09:42:25 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get state
2023-04-19 09:42:25 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to set attempt sync config
2023-04-19 09:42:25 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/2/1/logs.log
2023-04-19 09:42:25 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.44.0
2023-04-19 09:42:25 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to save workflow id for cancellation
2023-04-19 09:42:25 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:25 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:25 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:25 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- START CHECK -----
2023-04-19 09:42:25 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:25 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:25 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:25 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:25 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:25 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/source-customize-api:0.0.1 exists...
2023-04-19 09:42:25 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/source-customize-api:0.0.1 was found locally.
2023-04-19 09:42:25 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = source-customize-api-check-2-1-hwhds with resources io.airbyte.config.ResourceRequirements@7271a2f9[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:25 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/1 --log-driver none --name source-customize-api-check-2-1-hwhds --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/source-customize-api:0.0.1 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=1 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/source-customize-api:0.0.1 check --config source_config.json
2023-04-19 09:42:25 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:26 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Attempting to connect to API at http://192.168.100.158:9098/category/build/1...
2023-04-19 09:42:27 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Response from http://192.168.100.158:9098/category/build/1: {'status': 0, 'message': None, 'data': [{'id': 3, 'name': '数据资产分类', 'description': '数据资产分类', 'parentId': None, 'idPath': '3', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:17', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:17', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 4, 'name': '业务域', 'description': '业务域', 'parentId': 3, 'idPath': '3,4', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:17', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:17', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 5, 'name': '数据域', 'description': '数据域', 'parentId': 3, 'idPath': '3,5', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:17', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:17', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 6, 'name': '更新频率', 'description': '更新频率', 'parentId': 3, 'idPath': '3,6', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:17', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:17', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 7, 'name': '分析主题', 'description': '分析主题', 'parentId': 3, 'idPath': '3,7', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 8, 'name': '自定义', 'description': '自定义', 'parentId': 3, 'idPath': '3,8', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 9, 'name': '组织', 'description': '组织', 'parentId': None, 'idPath': '9', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 10, 'name': '一级部门', 'description': '一级部门', 'parentId': 9, 'idPath': '9,10', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 11, 'name': '二级部门', 'description': '二级部门', 'parentId': 10, 'idPath': '9,10,11', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 12, 'name': '数据资产分类2', 'description': '数据资产分类', 'parentId': None, 'idPath': '12', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 13, 'name': '业务域2', 'description': '业务域', 'parentId': 12, 'idPath': '12,13', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 14, 'name': '数据域2', 'description': '数据域', 'parentId': 12, 'idPath': '12,14', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 15, 'name': '更新频率2', 'description': '更新频率', 'parentId': 12, 'idPath': '12,15', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 16, 'name': '分析主题2', 'description': '分析主题', 'parentId': 12, 'idPath': '12,16', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 17, 'name': '自定义2', 'description': '自定义', 'parentId': 12, 'idPath': '12,17', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 18, 'name': '组织2', 'description': '组织', 'parentId': None, 'idPath': '18', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 19, 'name': '一级部门2', 'description': '一级部门', 'parentId': 18, 'idPath': '18,19', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 20, 'name': '二级部门2', 'description': '二级部门', 'parentId': 19, 'idPath': '18,19,20', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 21, 'name': '数据资产分类3', 'description': '数据资产分类', 'parentId': None, 'idPath': '21', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 22, 'name': '业务域3', 'description': '业务域', 'parentId': 21, 'idPath': '21,22', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 23, 'name': '数据域3', 'description': '数据域', 'parentId': 21, 'idPath': '21,23', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 24, 'name': '更新频率3', 'description': '更新频率', 'parentId': 21, 'idPath': '21,24', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 25, 'name': '分析主题3', 'description': '分析主题', 'parentId': 21, 'idPath': '21,25', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 26, 'name': '自定义3', 'description': '自定义', 'parentId': 21, 'idPath': '21,26', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 27, 'name': '组织3', 'description': '组织', 'parentId': None, 'idPath': '27', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 28, 'name': '一级部门3', 'description': '一级部门', 'parentId': 27, 'idPath': '27,28', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 29, 'name': '二级部门3', 'description': '二级部门', 'parentId': 28, 'idPath': '27,28,29', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 30, 'name': '数据资产分类4', 'description': '数据资产分类', 'parentId': None, 'idPath': '30', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 31, 'name': '业务域4', 'description': '业务域', 'parentId': 30, 'idPath': '30,31', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 32, 'name': '数据域4', 'description': '数据域', 'parentId': 30, 'idPath': '30,32', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 33, 'name': '更新频率4', 'description': '更新频率', 'parentId': 30, 'idPath': '30,33', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 34, 'name': '分析主题4', 'description': '分析主题', 'parentId': 30, 'idPath': '30,34', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 35, 'name': '自定义4', 'description': '自定义', 'parentId': 30, 'idPath': '30,35', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 36, 'name': '组织4', 'description': '组织', 'parentId': None, 'idPath': '36', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 37, 'name': '一级部门4', 'description': '一级部门', 'parentId': 36, 'idPath': '36,37', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 38, 'name': '二级部门4', 'description': '二级部门', 'parentId': 37, 'idPath': '36,37,38', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 39, 'name': '数据资产分类5', 'description': '数据资产分类', 'parentId': None, 'idPath': '39', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 40, 'name': '业务域5', 'description': '业务域', 'parentId': 39, 'idPath': '39,40', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 41, 'name': '数据域5', 'description': '数据域', 'parentId': 39, 'idPath': '39,41', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 42, 'name': '更新频率5', 'description': '更新频率', 'parentId': 39, 'idPath': '39,42', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 43, 'name': '分析主题5', 'description': '分析主题', 'parentId': 39, 'idPath': '39,43', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 44, 'name': '自定义5', 'description': '自定义', 'parentId': 39, 'idPath': '39,44', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 45, 'name': '组织5', 'description': '组织', 'parentId': None, 'idPath': '45', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 46, 'name': '一级部门5', 'description': '一级部门', 'parentId': 45, 'idPath': '45,46', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 47, 'name': '二级部门5', 'description': '二级部门', 'parentId': 46, 'idPath': '45,46,47', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 48, 'name': '数据资产分类6', 'description': '数据资产分类', 'parentId': None, 'idPath': '48', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:50', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:50', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 49, 'name': '业务域6', 'description': '业务域', 'parentId': 48, 'idPath': '48,49', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:50', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:50', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 50, 'name': '数据域6', 'description': '数据域', 'parentId': 48, 'idPath': '48,50', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:50', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:50', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 51, 'name': '更新频率6', 'description': '更新频率', 'parentId': 48, 'idPath': '48,51', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:50', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:50', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 52, 'name': '分析主题6', 'description': '分析主题', 'parentId': 48, 'idPath': '48,52', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 53, 'name': '自定义6', 'description': '自定义', 'parentId': 48, 'idPath': '48,53', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 54, 'name': '组织6', 'description': '组织', 'parentId': None, 'idPath': '54', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 55, 'name': '一级部门6', 'description': '一级部门', 'parentId': 54, 'idPath': '54,55', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 56, 'name': '二级部门6', 'description': '二级部门', 'parentId': 55, 'idPath': '54,55,56', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 66, 'name': '数据资产分类7', 'description': '数据资产分类', 'parentId': None, 'idPath': '66', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 67, 'name': '业务域7', 'description': '业务域', 'parentId': 66, 'idPath': '66,67', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 68, 'name': '数据域7', 'description': '数据域', 'parentId': 66, 'idPath': '66,68', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 69, 'name': '更新频率7', 'description': '更新频率', 'parentId': 66, 'idPath': '66,69', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 70, 'name': '分析主题7', 'description': '分析主题', 'parentId': 66, 'idPath': '66,70', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 71, 'name': '自定义7', 'description': '自定义', 'parentId': 66, 'idPath': '66,71', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 72, 'name': '组织7', 'description': '组织', 'parentId': None, 'idPath': '72', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 73, 'name': '一级部门7', 'description': '一级部门', 'parentId': 72, 'idPath': '72,73', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 74, 'name': '二级部门7', 'description': '二级部门', 'parentId': 73, 'idPath': '72,73,74', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 156, 'name': '组织8', 'description': '', 'parentId': None, 'idPath': '156', 'params': None, 'systems': 'DDP', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-04-04T14:20:35', 'createBy': 'qa25org_admin', 'updateTime': '2023-04-04T14:20:35', 'updateBy': 'qa25org_admin', 'tagCount': 0, 'children': None, 'scopes': []}], 'error': None}
2023-04-19 09:42:27 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Check connection status: True
2023-04-19 09:42:27 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Check connection success
2023-04-19 09:42:27 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Check succeeded
2023-04-19 09:42:27 [32mINFO[m i.a.w.g.DefaultCheckConnectionWorker(run):115 - Check connection job received output: io.airbyte.config.StandardCheckConnectionOutput@35b3c9c2[status=succeeded,message=<null>,additionalProperties={}]
2023-04-19 09:42:27 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-04-19 09:42:27 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:27 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- END CHECK -----
2023-04-19 09:42:27 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:27 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/2/1/logs.log
2023-04-19 09:42:27 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.44.0
2023-04-19 09:42:27 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to save workflow id for cancellation
2023-04-19 09:42:27 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:27 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:27 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:27 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:27 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:27 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:27 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- START CHECK -----
2023-04-19 09:42:27 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:27 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:27 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/destination-mysql:0.1.20 exists...
2023-04-19 09:42:27 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/destination-mysql:0.1.20 was found locally.
2023-04-19 09:42:27 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = destination-mysql-check-2-1-ejpqy with resources io.airbyte.config.ResourceRequirements@7271a2f9[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:27 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/1 --log-driver none --name destination-mysql-check-2-1-ejpqy --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/destination-mysql:0.1.20 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=1 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/destination-mysql:0.1.20 check --config source_config.json
2023-04-19 09:42:27 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:27 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Class path contains multiple SLF4J bindings.
2023-04-19 09:42:27 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:27 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:27 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Found binding in [jar:file:/airbyte/lib/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:27 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
2023-04-19 09:42:27 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):136 - starting destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationCliParser(parseOptions):118 - integration args: {check=null, config=source_config.json}"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):123 - Running integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):124 - Command: CHECK"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):125 - Integration config: IntegrationConfig{command=CHECK, configPath='source_config.json', catalogPath='null', statePath='null'}"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword order - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword examples - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword multiline - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m i.a.i.b.s.SshTunnel(getInstance):170 - Starting connection with method: NO_TUNNEL"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):80 - HikariPool-1 - Starting..."
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):82 - HikariPool-1 - Start completed."
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(close):350 - HikariPool-1 - Shutdown initiated..."
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(close):352 - HikariPool-1 - Shutdown completed."
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):171 - Completed integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:28 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:28 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):138 - completed destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:28 [32mINFO[m i.a.w.g.DefaultCheckConnectionWorker(run):115 - Check connection job received output: io.airbyte.config.StandardCheckConnectionOutput@1d74694f[status=succeeded,message=<null>,additionalProperties={}]
2023-04-19 09:42:28 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-04-19 09:42:28 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:28 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- END CHECK -----
2023-04-19 09:42:28 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:28 [32mINFO[m i.a.c.t.s.DefaultTaskQueueMapper(getTaskQueue):31 - Called DefaultTaskQueueMapper getTaskQueue for geography auto
2023-04-19 09:42:29 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/2/1/logs.log
2023-04-19 09:42:29 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.44.0
2023-04-19 09:42:29 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to save workflow id for cancellation
2023-04-19 09:42:29 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get the source for heartbeat
2023-04-19 09:42:29 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get the source definition
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):93 - Setting up source...
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):97 - Setting up destination...
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable METRIC_CLIENT: ''
2023-04-19 09:42:29 [33mWARN[m i.a.m.l.MetricClientFactory(initialize):60 - Metric client is already initialized to 
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):106 - Setting up replication worker...
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):161 - start sync worker. job id: 2 attempt id: 1
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):163 - Committing states from replication activity
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):166 - Committing stats from replication activity
2023-04-19 09:42:29 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):182 - configured sync modes: {null.my_stream=full_refresh - overwrite}
2023-04-19 09:42:29 [32mINFO[m i.a.w.i.DefaultAirbyteDestination(start):92 - Running destination...
2023-04-19 09:42:29 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- START REPLICATION -----
2023-04-19 09:42:29 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:29 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/destination-mysql:0.1.20 exists...
2023-04-19 09:42:29 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/destination-mysql:0.1.20 was found locally.
2023-04-19 09:42:29 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = destination-mysql-write-2-1-ybrid with resources io.airbyte.config.ResourceRequirements@59747b58[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:29 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/1 --log-driver none --name destination-mysql-write-2-1-ybrid --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/destination-mysql:0.1.20 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=1 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/destination-mysql:0.1.20 write --config destination_config.json --catalog destination_catalog.json
2023-04-19 09:42:29 [32mINFO[m i.a.w.i.VersionedAirbyteMessageBufferedWriterFactory(createWriter):41 - Writing messages to protocol version 0.2.0
2023-04-19 09:42:29 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:29 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:29 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/source-customize-api:0.0.1 exists...
2023-04-19 09:42:29 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/source-customize-api:0.0.1 was found locally.
2023-04-19 09:42:29 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = source-customize-api-read-2-1-tumbc with resources io.airbyte.config.ResourceRequirements@17220d6f[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:29 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/1 --log-driver none --name source-customize-api-read-2-1-tumbc --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/source-customize-api:0.0.1 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=1 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/source-customize-api:0.0.1 read --config source_config.json --catalog source_catalog.json
2023-04-19 09:42:29 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:29 [32mINFO[m i.a.w.i.HeartbeatTimeoutChaperone(runWithHeartbeatThread):94 - Starting source heartbeat check. Will check every 1 minutes.
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$8):376 - Replication thread started.
2023-04-19 09:42:29 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromDstRunnable$4):310 - Destination output thread started.
2023-04-19 09:42:29 [43mdestination[0m > SLF4J: Class path contains multiple SLF4J bindings.
2023-04-19 09:42:29 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:29 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:29 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:29 [43mdestination[0m > SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
2023-04-19 09:42:29 [43mdestination[0m > SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
2023-04-19 09:42:29 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:29 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):136 - starting destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationCliParser(parseOptions):118 - integration args: {catalog=destination_catalog.json, write=null, config=destination_config.json}"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):123 - Running integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):124 - Command: WRITE"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):125 - Integration config: IntegrationConfig{command=WRITE, configPath='destination_config.json', catalogPath='destination_catalog.json', statePath='null'}"
2023-04-19 09:42:30 [44msource[0m > Starting syncing SourceCustomizeAPI
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword order - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword examples - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword multiline - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:30 [44msource[0m > {"status":500,"message":"用户信息不能为空","data":null,"error":null}
2023-04-19 09:42:30 [44msource[0m > 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 74, in handle_http_error
    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]
KeyError: 401

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/airbyte/integration_code/main.py", line 13, in <module>
    launch(source, sys.argv[1:])
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 131, in launch
    for message in source_entrypoint.run(parsed_args):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 122, in run
    for message in generator:
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py", line 110, in read
    stream_is_available, error = stream_instance.check_availability(logger, self)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py", line 190, in check_availability
    return self.availability_strategy.check_availability(self, logger, source)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 51, in check_availability
    return self.handle_http_error(stream, logger, source, error)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 81, in handle_http_error
    raise error
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 45, in check_availability
    get_first_record_for_slice(stream, stream_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py", line 38, in get_first_record_for_slice
    return next(records_for_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 423, in read_records
    yield from self._read_pages(
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 439, in _read_pages
    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 462, in _fetch_next_page
    response = self._send_request(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 364, in _send_request
    return backoff_handler(user_backoff_handler)(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 331, in _send
    raise exc
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 328, in _send
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
2023-04-19 09:42:30 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$8):438 - Source has no more messages, closing connection.
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.b.s.SshTunnel(getInstance):170 - Starting connection with method: NO_TUNNEL"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):80 - HikariPool-1 - Starting..."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):82 - HikariPool-1 - Start completed."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$toWriteConfig$0):98 - Write config: WriteConfig{streamName=my_stream, namespace=null, outputSchemaName=api, tmpTableName=_airbyte_tmp_kzs_my_stream, outputTableName=_airbyte_raw_my_stream, syncMode=overwrite}"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.b.BufferedStreamConsumer(startTracked):116 - class io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer started."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):123 - Preparing tmp tables in destination started for 1 streams"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):127 - Preparing tmp table in destination started for stream my_stream. schema: api, tmp table name: _airbyte_tmp_kzs_my_stream"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):133 - Preparing tables in destination completed."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.b.FailureTrackingAirbyteMessageConsumer(close):65 - Airbyte message consumer: succeeded."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.b.BufferedStreamConsumer(close):171 - executing on success close procedure."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.SqlOperations(onDestinationCloseOperations):138 - No onDestinationCloseOperations required for this destination."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):163 - Finalizing tables in destination started for 1 streams"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):168 - Finalizing stream my_stream. schema api, tmp table _airbyte_tmp_kzs_my_stream, final table _airbyte_raw_my_stream"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):181 - Executing finalization of tables."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):183 - Finalizing tables in destination completed."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):186 - Cleaning tmp tables in destination started for 1 streams"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):190 - Cleaning tmp table in destination started for stream my_stream. schema api, tmp table name: _airbyte_tmp_kzs_my_stream"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):195 - Cleaning tmp tables in destination completed."
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):171 - Completed integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:30 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):138 - completed destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:30 [1;31mERROR[m i.a.w.g.DefaultReplicationWorker(replicate):278 - Sync worker failed.
io.airbyte.workers.internal.exception.SourceException: Source didn't exit properly - check the logs!
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:442) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
	Suppressed: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.
		at io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.replicate(DefaultReplicationWorker.java:214) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.run(DefaultReplicationWorker.java:190) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.run(DefaultReplicationWorker.java:91) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$5(TemporalAttemptExecution.java:195) ~[io.airbyte-airbyte-workers-0.44.0.jar:?]
		at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.
	at io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:440) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
	... 4 more
2023-04-19 09:42:30 [32mINFO[m i.a.w.g.DefaultReplicationWorker(getReplicationOutput):561 - sync summary: {
  "status" : "failed",
  "recordsSynced" : 0,
  "bytesSynced" : 0,
  "startTime" : 1681897349075,
  "endTime" : 1681897350858,
  "totalStats" : {
    "bytesCommitted" : 0,
    "bytesEmitted" : 0,
    "destinationStateMessagesEmitted" : 0,
    "destinationWriteEndTime" : 1681897350857,
    "destinationWriteStartTime" : 1681897349173,
    "meanSecondsBeforeSourceStateMessageEmitted" : 0,
    "maxSecondsBeforeSourceStateMessageEmitted" : 0,
    "maxSecondsBetweenStateMessageEmittedandCommitted" : 0,
    "meanSecondsBetweenStateMessageEmittedandCommitted" : 0,
    "recordsEmitted" : 0,
    "recordsCommitted" : 0,
    "replicationEndTime" : 1681897350858,
    "replicationStartTime" : 1681897349075,
    "sourceReadEndTime" : 0,
    "sourceReadStartTime" : 1681897349117,
    "sourceStateMessagesEmitted" : 0
  },
  "streamStats" : [ ]
}
2023-04-19 09:42:30 [32mINFO[m i.a.w.g.DefaultReplicationWorker(getReplicationOutput):562 - failures: [ {
  "failureOrigin" : "source",
  "failureType" : "system_error",
  "internalMessage" : "401 Client Error:  for url: http://192.168.100.158:9098/category/build/1",
  "externalMessage" : "Something went wrong in the connector. See the logs for more details.",
  "metadata" : {
    "attemptNumber" : 1,
    "jobId" : 2,
    "from_trace_message" : true,
    "connector_command" : "read"
  },
  "stacktrace" : "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 74, in handle_http_error\n    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]\nKeyError: 401\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/airbyte/integration_code/main.py\", line 13, in <module>\n    launch(source, sys.argv[1:])\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 131, in launch\n    for message in source_entrypoint.run(parsed_args):\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 122, in run\n    for message in generator:\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 110, in read\n    stream_is_available, error = stream_instance.check_availability(logger, self)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py\", line 190, in check_availability\n    return self.availability_strategy.check_availability(self, logger, source)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 51, in check_availability\n    return self.handle_http_error(stream, logger, source, error)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 81, in handle_http_error\n    raise error\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 45, in check_availability\n    get_first_record_for_slice(stream, stream_slice)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py\", line 38, in get_first_record_for_slice\n    return next(records_for_slice)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 423, in read_records\n    yield from self._read_pages(\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 439, in _read_pages\n    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 462, in _fetch_next_page\n    response = self._send_request(request, request_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 364, in _send_request\n    return backoff_handler(user_backoff_handler)(request, request_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/backoff/_sync.py\", line 105, in retry\n    ret = target(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/backoff/_sync.py\", line 105, in retry\n    ret = target(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 331, in _send\n    raise exc\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 328, in _send\n    response.raise_for_status()\n  File \"/usr/local/lib/python3.9/site-packages/requests/models.py\", line 1021, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1\n",
  "timestamp" : 1681897350222
}, {
  "failureOrigin" : "source",
  "internalMessage" : "Source didn't exit properly - check the logs!",
  "externalMessage" : "Something went wrong within the source connector",
  "metadata" : {
    "attemptNumber" : 1,
    "jobId" : 2,
    "connector_command" : "read"
  },
  "stacktrace" : "io.airbyte.workers.internal.exception.SourceException: Source didn't exit properly - check the logs!\n\tat io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:442)\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1589)\nCaused by: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.\n\tat io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150)\n\tat io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:440)\n\t... 4 more\n",
  "timestamp" : 1681897350371
} ]
2023-04-19 09:42:30 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:30 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- END REPLICATION -----
2023-04-19 09:42:30 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:30 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-04-19 09:42:30 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$3):197 - sync summary: io.airbyte.config.StandardSyncOutput@6423db4e[standardSyncSummary=io.airbyte.config.StandardSyncSummary@46466758[status=failed,recordsSynced=0,bytesSynced=0,startTime=1681897349075,endTime=1681897350858,totalStats=io.airbyte.config.SyncStats@6a88e6ce[bytesCommitted=0,bytesEmitted=0,destinationStateMessagesEmitted=0,destinationWriteEndTime=1681897350857,destinationWriteStartTime=1681897349173,estimatedBytes=<null>,estimatedRecords=<null>,meanSecondsBeforeSourceStateMessageEmitted=0,maxSecondsBeforeSourceStateMessageEmitted=0,maxSecondsBetweenStateMessageEmittedandCommitted=0,meanSecondsBetweenStateMessageEmittedandCommitted=0,recordsEmitted=0,recordsCommitted=0,replicationEndTime=1681897350858,replicationStartTime=1681897349075,sourceReadEndTime=0,sourceReadStartTime=1681897349117,sourceStateMessagesEmitted=0,additionalProperties={}],streamStats=[],additionalProperties={}],normalizationSummary=<null>,webhookOperationSummary=<null>,state=<null>,outputCatalog=io.airbyte.protocol.models.ConfiguredAirbyteCatalog@6767aefa[streams=[io.airbyte.protocol.models.ConfiguredAirbyteStream@1de4e0f3[stream=io.airbyte.protocol.models.AirbyteStream@686d28b0[name=my_stream,jsonSchema={"$schema":"http://json-schema.org/draft-07/schema","type":"object","required":["url"],"properties":{"url":{"description":"The URL for the API endpoint.","type":"string"},"token":{"description":"The API token for authentication.","type":"string"}}},supportedSyncModes=[full_refresh],sourceDefinedCursor=<null>,defaultCursorField=[],sourceDefinedPrimaryKey=[],namespace=<null>,additionalProperties={}],syncMode=full_refresh,cursorField=[],destinationSyncMode=overwrite,primaryKey=[],additionalProperties={}]],additionalProperties={}],failures=[io.airbyte.config.FailureReason@457e0de5[failureOrigin=source,failureType=system_error,internalMessage=401 Client Error:  for url: http://192.168.100.158:9098/category/build/1,externalMessage=Something went wrong in the connector. See the logs for more details.,metadata=io.airbyte.config.Metadata@1fb47bce[additionalProperties={attemptNumber=1, jobId=2, from_trace_message=true, connector_command=read}],stacktrace=Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 74, in handle_http_error
    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]
KeyError: 401

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/airbyte/integration_code/main.py", line 13, in <module>
    launch(source, sys.argv[1:])
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 131, in launch
    for message in source_entrypoint.run(parsed_args):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 122, in run
    for message in generator:
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py", line 110, in read
    stream_is_available, error = stream_instance.check_availability(logger, self)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py", line 190, in check_availability
    return self.availability_strategy.check_availability(self, logger, source)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 51, in check_availability
    return self.handle_http_error(stream, logger, source, error)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 81, in handle_http_error
    raise error
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 45, in check_availability
    get_first_record_for_slice(stream, stream_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py", line 38, in get_first_record_for_slice
    return next(records_for_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 423, in read_records
    yield from self._read_pages(
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 439, in _read_pages
    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 462, in _fetch_next_page
    response = self._send_request(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 364, in _send_request
    return backoff_handler(user_backoff_handler)(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 331, in _send
    raise exc
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 328, in _send
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
,retryable=<null>,timestamp=1681897350222,additionalProperties={}], io.airbyte.config.FailureReason@25ede3b3[failureOrigin=source,failureType=<null>,internalMessage=Source didn't exit properly - check the logs!,externalMessage=Something went wrong within the source connector,metadata=io.airbyte.config.Metadata@24c47716[additionalProperties={attemptNumber=1, jobId=2, connector_command=read}],stacktrace=io.airbyte.workers.internal.exception.SourceException: Source didn't exit properly - check the logs!
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:442)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1589)
Caused by: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.
	at io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150)
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:440)
	... 4 more
,retryable=<null>,timestamp=1681897350371,additionalProperties={}]],commitStateAsap=true,additionalProperties={}]
2023-04-19 09:42:30 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$3):202 - Sync summary length: 6951
2023-04-19 09:42:30 [32mINFO[m i.a.c.t.TemporalUtils(withBackgroundHeartbeat):307 - Stopping temporal heartbeating...
2023-04-19 09:42:34 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to Get a connection by connection Id
2023-04-19 09:42:34 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get the most recent source actor catalog
2023-04-19 09:42:34 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to Get a connection by connection Id
2023-04-19 09:42:31 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get state
2023-04-19 09:42:31 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to set attempt sync config
2023-04-19 09:42:31 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/2/2/logs.log
2023-04-19 09:42:31 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.44.0
2023-04-19 09:42:31 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to save workflow id for cancellation
2023-04-19 09:42:31 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:31 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:31 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- START CHECK -----
2023-04-19 09:42:31 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:31 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:31 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:31 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:31 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:31 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:31 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/source-customize-api:0.0.1 exists...
2023-04-19 09:42:31 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/source-customize-api:0.0.1 was found locally.
2023-04-19 09:42:31 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = source-customize-api-check-2-2-ptusl with resources io.airbyte.config.ResourceRequirements@7271a2f9[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:31 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/2 --log-driver none --name source-customize-api-check-2-2-ptusl --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/source-customize-api:0.0.1 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=2 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/source-customize-api:0.0.1 check --config source_config.json
2023-04-19 09:42:31 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:32 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Attempting to connect to API at http://192.168.100.158:9098/category/build/1...
2023-04-19 09:42:32 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Response from http://192.168.100.158:9098/category/build/1: {'status': 0, 'message': None, 'data': [{'id': 3, 'name': '数据资产分类', 'description': '数据资产分类', 'parentId': None, 'idPath': '3', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:17', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:17', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 4, 'name': '业务域', 'description': '业务域', 'parentId': 3, 'idPath': '3,4', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:17', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:17', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 5, 'name': '数据域', 'description': '数据域', 'parentId': 3, 'idPath': '3,5', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:17', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:17', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 6, 'name': '更新频率', 'description': '更新频率', 'parentId': 3, 'idPath': '3,6', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:17', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:17', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 7, 'name': '分析主题', 'description': '分析主题', 'parentId': 3, 'idPath': '3,7', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 8, 'name': '自定义', 'description': '自定义', 'parentId': 3, 'idPath': '3,8', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 9, 'name': '组织', 'description': '组织', 'parentId': None, 'idPath': '9', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 10, 'name': '一级部门', 'description': '一级部门', 'parentId': 9, 'idPath': '9,10', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 11, 'name': '二级部门', 'description': '二级部门', 'parentId': 10, 'idPath': '9,10,11', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:54:18', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:54:18', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 12, 'name': '数据资产分类2', 'description': '数据资产分类', 'parentId': None, 'idPath': '12', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 13, 'name': '业务域2', 'description': '业务域', 'parentId': 12, 'idPath': '12,13', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 14, 'name': '数据域2', 'description': '数据域', 'parentId': 12, 'idPath': '12,14', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 15, 'name': '更新频率2', 'description': '更新频率', 'parentId': 12, 'idPath': '12,15', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 16, 'name': '分析主题2', 'description': '分析主题', 'parentId': 12, 'idPath': '12,16', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 17, 'name': '自定义2', 'description': '自定义', 'parentId': 12, 'idPath': '12,17', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 18, 'name': '组织2', 'description': '组织', 'parentId': None, 'idPath': '18', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 19, 'name': '一级部门2', 'description': '一级部门', 'parentId': 18, 'idPath': '18,19', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 20, 'name': '二级部门2', 'description': '二级部门', 'parentId': 19, 'idPath': '18,19,20', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T16:55:01', 'createBy': 'brandon', 'updateTime': '2023-03-13T16:55:01', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 21, 'name': '数据资产分类3', 'description': '数据资产分类', 'parentId': None, 'idPath': '21', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 22, 'name': '业务域3', 'description': '业务域', 'parentId': 21, 'idPath': '21,22', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 23, 'name': '数据域3', 'description': '数据域', 'parentId': 21, 'idPath': '21,23', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 24, 'name': '更新频率3', 'description': '更新频率', 'parentId': 21, 'idPath': '21,24', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 25, 'name': '分析主题3', 'description': '分析主题', 'parentId': 21, 'idPath': '21,25', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 26, 'name': '自定义3', 'description': '自定义', 'parentId': 21, 'idPath': '21,26', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 27, 'name': '组织3', 'description': '组织', 'parentId': None, 'idPath': '27', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 28, 'name': '一级部门3', 'description': '一级部门', 'parentId': 27, 'idPath': '27,28', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 29, 'name': '二级部门3', 'description': '二级部门', 'parentId': 28, 'idPath': '27,28,29', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:11:20', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:11:20', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 30, 'name': '数据资产分类4', 'description': '数据资产分类', 'parentId': None, 'idPath': '30', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 31, 'name': '业务域4', 'description': '业务域', 'parentId': 30, 'idPath': '30,31', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 32, 'name': '数据域4', 'description': '数据域', 'parentId': 30, 'idPath': '30,32', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 33, 'name': '更新频率4', 'description': '更新频率', 'parentId': 30, 'idPath': '30,33', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 34, 'name': '分析主题4', 'description': '分析主题', 'parentId': 30, 'idPath': '30,34', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 35, 'name': '自定义4', 'description': '自定义', 'parentId': 30, 'idPath': '30,35', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 36, 'name': '组织4', 'description': '组织', 'parentId': None, 'idPath': '36', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 37, 'name': '一级部门4', 'description': '一级部门', 'parentId': 36, 'idPath': '36,37', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 38, 'name': '二级部门4', 'description': '二级部门', 'parentId': 37, 'idPath': '36,37,38', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:31', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:31', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 39, 'name': '数据资产分类5', 'description': '数据资产分类', 'parentId': None, 'idPath': '39', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 40, 'name': '业务域5', 'description': '业务域', 'parentId': 39, 'idPath': '39,40', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 41, 'name': '数据域5', 'description': '数据域', 'parentId': 39, 'idPath': '39,41', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 42, 'name': '更新频率5', 'description': '更新频率', 'parentId': 39, 'idPath': '39,42', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 43, 'name': '分析主题5', 'description': '分析主题', 'parentId': 39, 'idPath': '39,43', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 44, 'name': '自定义5', 'description': '自定义', 'parentId': 39, 'idPath': '39,44', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 45, 'name': '组织5', 'description': '组织', 'parentId': None, 'idPath': '45', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 46, 'name': '一级部门5', 'description': '一级部门', 'parentId': 45, 'idPath': '45,46', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 47, 'name': '二级部门5', 'description': '二级部门', 'parentId': 46, 'idPath': '45,46,47', 'params': None, 'systems': 'DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:42', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:42', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 48, 'name': '数据资产分类6', 'description': '数据资产分类', 'parentId': None, 'idPath': '48', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:50', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:50', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 49, 'name': '业务域6', 'description': '业务域', 'parentId': 48, 'idPath': '48,49', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:50', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:50', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 50, 'name': '数据域6', 'description': '数据域', 'parentId': 48, 'idPath': '48,50', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:50', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:50', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 51, 'name': '更新频率6', 'description': '更新频率', 'parentId': 48, 'idPath': '48,51', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:50', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:50', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 52, 'name': '分析主题6', 'description': '分析主题', 'parentId': 48, 'idPath': '48,52', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 53, 'name': '自定义6', 'description': '自定义', 'parentId': 48, 'idPath': '48,53', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 54, 'name': '组织6', 'description': '组织', 'parentId': None, 'idPath': '54', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 55, 'name': '一级部门6', 'description': '一级部门', 'parentId': 54, 'idPath': '54,55', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 56, 'name': '二级部门6', 'description': '二级部门', 'parentId': 55, 'idPath': '54,55,56', 'params': None, 'systems': 'Flowman', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:16:51', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:16:51', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 66, 'name': '数据资产分类7', 'description': '数据资产分类', 'parentId': None, 'idPath': '66', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 67, 'name': '业务域7', 'description': '业务域', 'parentId': 66, 'idPath': '66,67', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 68, 'name': '数据域7', 'description': '数据域', 'parentId': 66, 'idPath': '66,68', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 69, 'name': '更新频率7', 'description': '更新频率', 'parentId': 66, 'idPath': '66,69', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 70, 'name': '分析主题7', 'description': '分析主题', 'parentId': 66, 'idPath': '66,70', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}, {'id': 71, 'name': '自定义7', 'description': '自定义', 'parentId': 66, 'idPath': '66,71', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}, {'id': 72, 'name': '组织7', 'description': '组织', 'parentId': None, 'idPath': '72', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 73, 'name': '一级部门7', 'description': '一级部门', 'parentId': 72, 'idPath': '72,73', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': [{'id': 74, 'name': '二级部门7', 'description': '二级部门', 'parentId': 73, 'idPath': '72,73,74', 'params': None, 'systems': 'DDP,DAAM', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-03-13T17:20:40', 'createBy': 'brandon', 'updateTime': '2023-03-13T17:20:40', 'updateBy': 'brandon', 'tagCount': 0, 'children': None, 'scopes': []}], 'scopes': []}], 'scopes': []}, {'id': 156, 'name': '组织8', 'description': '', 'parentId': None, 'idPath': '156', 'params': None, 'systems': 'DDP', 'sort': 0, 'type': 1, 'status': 0, 'namespace': None, 'groupId': '0a71e4b0-c3a1-11ec-96bd-35126eddec60', 'groupName': 'qa25org', 'createTime': '2023-04-04T14:20:35', 'createBy': 'qa25org_admin', 'updateTime': '2023-04-04T14:20:35', 'updateBy': 'qa25org_admin', 'tagCount': 0, 'children': None, 'scopes': []}], 'error': None}
2023-04-19 09:42:32 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Check connection status: True
2023-04-19 09:42:32 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Check connection success
2023-04-19 09:42:32 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(internalLog):317 - Check succeeded
2023-04-19 09:42:32 [32mINFO[m i.a.w.g.DefaultCheckConnectionWorker(run):115 - Check connection job received output: io.airbyte.config.StandardCheckConnectionOutput@47d95b89[status=succeeded,message=<null>,additionalProperties={}]
2023-04-19 09:42:32 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-04-19 09:42:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- END CHECK -----
2023-04-19 09:42:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:32 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/2/2/logs.log
2023-04-19 09:42:32 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.44.0
2023-04-19 09:42:32 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to save workflow id for cancellation
2023-04-19 09:42:32 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- START CHECK -----
2023-04-19 09:42:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:32 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:32 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:32 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:32 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:32 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/destination-mysql:0.1.20 exists...
2023-04-19 09:42:32 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/destination-mysql:0.1.20 was found locally.
2023-04-19 09:42:32 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = destination-mysql-check-2-2-gzypr with resources io.airbyte.config.ResourceRequirements@7271a2f9[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:32 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/2 --log-driver none --name destination-mysql-check-2-2-gzypr --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/destination-mysql:0.1.20 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=2 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/destination-mysql:0.1.20 check --config source_config.json
2023-04-19 09:42:32 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:33 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Class path contains multiple SLF4J bindings.
2023-04-19 09:42:33 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:33 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:33 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Found binding in [jar:file:/airbyte/lib/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:33 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
2023-04-19 09:42:33 [1;31mERROR[m i.a.c.i.LineGobbler(voidCall):149 - SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):136 - starting destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationCliParser(parseOptions):118 - integration args: {check=null, config=source_config.json}"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):123 - Running integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):124 - Command: CHECK"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):125 - Integration config: IntegrationConfig{command=CHECK, configPath='source_config.json', catalogPath='null', statePath='null'}"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword order - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword examples - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword multiline - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[32mINFO\u001B[m i.a.i.b.s.SshTunnel(getInstance):170 - Starting connection with method: NO_TUNNEL"
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):80 - HikariPool-1 - Starting..."
2023-04-19 09:42:33 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:33 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):82 - HikariPool-1 - Start completed."
2023-04-19 09:42:34 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:34 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(close):350 - HikariPool-1 - Shutdown initiated..."
2023-04-19 09:42:34 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:34 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(close):352 - HikariPool-1 - Shutdown completed."
2023-04-19 09:42:34 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:34 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):171 - Completed integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:34 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:34 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):138 - completed destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.DefaultCheckConnectionWorker(run):115 - Check connection job received output: io.airbyte.config.StandardCheckConnectionOutput@5da226d5[status=succeeded,message=<null>,additionalProperties={}]
2023-04-19 09:42:34 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- END CHECK -----
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:34 [32mINFO[m i.a.c.t.s.DefaultTaskQueueMapper(getTaskQueue):31 - Called DefaultTaskQueueMapper getTaskQueue for geography auto
2023-04-19 09:42:34 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):136 - Docker volume job log path: /tmp/workspace/2/2/logs.log
2023-04-19 09:42:34 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):141 - Executing worker wrapper. Airbyte version: 0.44.0
2023-04-19 09:42:34 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to save workflow id for cancellation
2023-04-19 09:42:34 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get the source for heartbeat
2023-04-19 09:42:34 [32mINFO[m i.a.a.c.AirbyteApiClient(retryWithJitter):179 - Attempt 0 to get the source definition
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):93 - Setting up source...
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):97 - Setting up destination...
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable METRIC_CLIENT: ''
2023-04-19 09:42:34 [33mWARN[m i.a.m.l.MetricClientFactory(initialize):60 - Metric client is already initialized to 
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.ReplicationWorkerFactory(create):106 - Setting up replication worker...
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):161 - start sync worker. job id: 2 attempt id: 2
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):163 - Committing states from replication activity
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):166 - Committing stats from replication activity
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- START REPLICATION -----
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.DefaultReplicationWorker(run):182 - configured sync modes: {null.my_stream=full_refresh - overwrite}
2023-04-19 09:42:34 [32mINFO[m i.a.w.i.DefaultAirbyteDestination(start):92 - Running destination...
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/destination-mysql:0.1.20 exists...
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/destination-mysql:0.1.20 was found locally.
2023-04-19 09:42:34 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = destination-mysql-write-2-2-fttxc with resources io.airbyte.config.ResourceRequirements@5117804b[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:34 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/2 --log-driver none --name destination-mysql-write-2-2-fttxc --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/destination-mysql:0.1.20 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=2 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/destination-mysql:0.1.20 write --config destination_config.json --catalog destination_catalog.json
2023-04-19 09:42:34 [32mINFO[m i.a.w.i.VersionedAirbyteMessageBufferedWriterFactory(createWriter):41 - Writing messages to protocol version 0.2.0
2023-04-19 09:42:34 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_LIMIT: '2.0'
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SIDECAR_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable SOCAT_KUBE_CPU_REQUEST: '0.1'
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable LAUNCHDARKLY_KEY: ''
2023-04-19 09:42:34 [32mINFO[m i.a.c.EnvConfigs(getEnvOrDefault):1222 - Using default value for environment variable FEATURE_FLAG_CLIENT: ''
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - Checking if airbyte/source-customize-api:0.0.1 exists...
2023-04-19 09:42:34 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - airbyte/source-customize-api:0.0.1 was found locally.
2023-04-19 09:42:34 [32mINFO[m i.a.w.p.DockerProcessFactory(create):133 - Creating docker container = source-customize-api-read-2-2-mamnk with resources io.airbyte.config.ResourceRequirements@508930b6[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=,additionalProperties={}] and allowedHosts null
2023-04-19 09:42:34 [32mINFO[m i.a.w.p.DockerProcessFactory(create):184 - Preparing command: docker run --rm --init -i -w /data/2/2 --log-driver none --name source-customize-api-read-2-2-mamnk --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e WORKER_CONNECTOR_IMAGE=airbyte/source-customize-api:0.0.1 -e AUTO_DETECT_SCHEMA=true -e LAUNCHDARKLY_KEY= -e SOCAT_KUBE_CPU_REQUEST=0.1 -e SOCAT_KUBE_CPU_LIMIT=2.0 -e USE_STREAM_CAPABLE_STATE=true -e FIELD_SELECTION_WORKSPACES= -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e APPLY_FIELD_SELECTION=false -e WORKER_JOB_ATTEMPT=2 -e FEATURE_FLAG_CLIENT= -e AIRBYTE_VERSION=0.44.0 -e WORKER_JOB_ID=2 airbyte/source-customize-api:0.0.1 read --config source_config.json --catalog source_catalog.json
2023-04-19 09:42:34 [32mINFO[m i.a.w.i.VersionedAirbyteStreamFactory(create):181 - Reading messages from protocol version 0.2.0
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromDstRunnable$4):310 - Destination output thread started.
2023-04-19 09:42:34 [32mINFO[m i.a.w.i.HeartbeatTimeoutChaperone(runWithHeartbeatThread):94 - Starting source heartbeat check. Will check every 1 minutes.
2023-04-19 09:42:34 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$8):376 - Replication thread started.
2023-04-19 09:42:34 [43mdestination[0m > SLF4J: Class path contains multiple SLF4J bindings.
2023-04-19 09:42:34 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:34 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:34 [43mdestination[0m > SLF4J: Found binding in [jar:file:/airbyte/lib/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2023-04-19 09:42:34 [43mdestination[0m > SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
2023-04-19 09:42:34 [43mdestination[0m > SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):136 - starting destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationCliParser(parseOptions):118 - integration args: {catalog=destination_catalog.json, write=null, config=destination_config.json}"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):123 - Running integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):124 - Command: WRITE"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):125 - Integration config: IntegrationConfig{command=WRITE, configPath='destination_config.json', catalogPath='destination_catalog.json', statePath='null'}"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword order - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword examples - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[33mWARN\u001B[m c.n.s.JsonMetaSchema(newValidator):338 - Unknown keyword multiline - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword"
2023-04-19 09:42:35 [44msource[0m > Starting syncing SourceCustomizeAPI
2023-04-19 09:42:35 [44msource[0m > {"status":500,"message":"用户信息不能为空","data":null,"error":null}
2023-04-19 09:42:35 [44msource[0m > 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 74, in handle_http_error
    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]
KeyError: 401

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/airbyte/integration_code/main.py", line 13, in <module>
    launch(source, sys.argv[1:])
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 131, in launch
    for message in source_entrypoint.run(parsed_args):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 122, in run
    for message in generator:
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py", line 110, in read
    stream_is_available, error = stream_instance.check_availability(logger, self)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py", line 190, in check_availability
    return self.availability_strategy.check_availability(self, logger, source)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 51, in check_availability
    return self.handle_http_error(stream, logger, source, error)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 81, in handle_http_error
    raise error
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 45, in check_availability
    get_first_record_for_slice(stream, stream_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py", line 38, in get_first_record_for_slice
    return next(records_for_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 423, in read_records
    yield from self._read_pages(
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 439, in _read_pages
    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 462, in _fetch_next_page
    response = self._send_request(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 364, in _send_request
    return backoff_handler(user_backoff_handler)(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 331, in _send
    raise exc
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 328, in _send
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.b.s.SshTunnel(getInstance):170 - Starting connection with method: NO_TUNNEL"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):80 - HikariPool-1 - Starting..."
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m c.z.h.HikariDataSource(<init>):82 - HikariPool-1 - Start completed."
2023-04-19 09:42:35 [32mINFO[m i.a.w.g.DefaultReplicationWorker(lambda$readFromSrcAndWriteToDstRunnable$8):438 - Source has no more messages, closing connection.
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$toWriteConfig$0):98 - Write config: WriteConfig{streamName=my_stream, namespace=null, outputSchemaName=api, tmpTableName=_airbyte_tmp_aqh_my_stream, outputTableName=_airbyte_raw_my_stream, syncMode=overwrite}"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.d.b.BufferedStreamConsumer(startTracked):116 - class io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer started."
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):123 - Preparing tmp tables in destination started for 1 streams"
2023-04-19 09:42:35 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:35 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):127 - Preparing tmp table in destination started for stream my_stream. schema: api, tmp table name: _airbyte_tmp_aqh_my_stream"
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onStartFunction$1):133 - Preparing tables in destination completed."
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.b.FailureTrackingAirbyteMessageConsumer(close):65 - Airbyte message consumer: succeeded."
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.b.BufferedStreamConsumer(close):171 - executing on success close procedure."
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.SqlOperations(onDestinationCloseOperations):138 - No onDestinationCloseOperations required for this destination."
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):163 - Finalizing tables in destination started for 1 streams"
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):168 - Finalizing stream my_stream. schema api, tmp table _airbyte_tmp_aqh_my_stream, final table _airbyte_raw_my_stream"
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):181 - Executing finalization of tables."
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):183 - Finalizing tables in destination completed."
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):186 - Cleaning tmp tables in destination started for 1 streams"
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):190 - Cleaning tmp table in destination started for stream my_stream. schema api, tmp table name: _airbyte_tmp_aqh_my_stream"
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.j.JdbcBufferedConsumerFactory(lambda$onCloseFunction$3):195 - Cleaning tmp tables in destination completed."
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.b.IntegrationRunner(runInternal):171 - Completed integration: io.airbyte.integrations.base.ssh.SshWrappedDestination"
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.i.VersionedAirbyteStreamFactory(toAirbyteMessage):360 - Deserialization failed: "2023-04-19 09:42:36 \u001B[32mINFO\u001B[m i.a.i.d.m.MySQLDestination(main):138 - completed destination: class io.airbyte.integrations.destination.mysql.MySQLDestination"
2023-04-19 09:42:36 [1;31mERROR[m i.a.w.g.DefaultReplicationWorker(replicate):278 - Sync worker failed.
io.airbyte.workers.internal.exception.SourceException: Source didn't exit properly - check the logs!
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:442) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
	Suppressed: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.
		at io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.replicate(DefaultReplicationWorker.java:214) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.run(DefaultReplicationWorker.java:190) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.general.DefaultReplicationWorker.run(DefaultReplicationWorker.java:91) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$5(TemporalAttemptExecution.java:195) ~[io.airbyte-airbyte-workers-0.44.0.jar:?]
		at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.
	at io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:440) ~[io.airbyte-airbyte-commons-worker-0.44.0.jar:?]
	... 4 more
2023-04-19 09:42:36 [32mINFO[m i.a.w.g.DefaultReplicationWorker(getReplicationOutput):561 - sync summary: {
  "status" : "failed",
  "recordsSynced" : 0,
  "bytesSynced" : 0,
  "startTime" : 1681897354487,
  "endTime" : 1681897356180,
  "totalStats" : {
    "bytesCommitted" : 0,
    "bytesEmitted" : 0,
    "destinationStateMessagesEmitted" : 0,
    "destinationWriteEndTime" : 1681897356179,
    "destinationWriteStartTime" : 1681897354616,
    "meanSecondsBeforeSourceStateMessageEmitted" : 0,
    "maxSecondsBeforeSourceStateMessageEmitted" : 0,
    "maxSecondsBetweenStateMessageEmittedandCommitted" : 0,
    "meanSecondsBetweenStateMessageEmittedandCommitted" : 0,
    "recordsEmitted" : 0,
    "recordsCommitted" : 0,
    "replicationEndTime" : 1681897356180,
    "replicationStartTime" : 1681897354487,
    "sourceReadEndTime" : 0,
    "sourceReadStartTime" : 1681897354529,
    "sourceStateMessagesEmitted" : 0
  },
  "streamStats" : [ ]
}
2023-04-19 09:42:36 [32mINFO[m i.a.w.g.DefaultReplicationWorker(getReplicationOutput):562 - failures: [ {
  "failureOrigin" : "source",
  "failureType" : "system_error",
  "internalMessage" : "401 Client Error:  for url: http://192.168.100.158:9098/category/build/1",
  "externalMessage" : "Something went wrong in the connector. See the logs for more details.",
  "metadata" : {
    "attemptNumber" : 2,
    "jobId" : 2,
    "from_trace_message" : true,
    "connector_command" : "read"
  },
  "stacktrace" : "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 74, in handle_http_error\n    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]\nKeyError: 401\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/airbyte/integration_code/main.py\", line 13, in <module>\n    launch(source, sys.argv[1:])\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 131, in launch\n    for message in source_entrypoint.run(parsed_args):\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 122, in run\n    for message in generator:\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 110, in read\n    stream_is_available, error = stream_instance.check_availability(logger, self)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py\", line 190, in check_availability\n    return self.availability_strategy.check_availability(self, logger, source)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 51, in check_availability\n    return self.handle_http_error(stream, logger, source, error)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 81, in handle_http_error\n    raise error\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py\", line 45, in check_availability\n    get_first_record_for_slice(stream, stream_slice)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py\", line 38, in get_first_record_for_slice\n    return next(records_for_slice)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 423, in read_records\n    yield from self._read_pages(\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 439, in _read_pages\n    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 462, in _fetch_next_page\n    response = self._send_request(request, request_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 364, in _send_request\n    return backoff_handler(user_backoff_handler)(request, request_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/backoff/_sync.py\", line 105, in retry\n    ret = target(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/backoff/_sync.py\", line 105, in retry\n    ret = target(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 331, in _send\n    raise exc\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py\", line 328, in _send\n    response.raise_for_status()\n  File \"/usr/local/lib/python3.9/site-packages/requests/models.py\", line 1021, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1\n",
  "timestamp" : 1681897355565
}, {
  "failureOrigin" : "source",
  "internalMessage" : "Source didn't exit properly - check the logs!",
  "externalMessage" : "Something went wrong within the source connector",
  "metadata" : {
    "attemptNumber" : 2,
    "jobId" : 2,
    "connector_command" : "read"
  },
  "stacktrace" : "io.airbyte.workers.internal.exception.SourceException: Source didn't exit properly - check the logs!\n\tat io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:442)\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1589)\nCaused by: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.\n\tat io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150)\n\tat io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:440)\n\t... 4 more\n",
  "timestamp" : 1681897355748
} ]
2023-04-19 09:42:36 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):163 - Stopping cancellation check scheduling...
2023-04-19 09:42:36 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:36 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - ----- END REPLICATION -----
2023-04-19 09:42:36 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$3):197 - sync summary: io.airbyte.config.StandardSyncOutput@6e3c34d5[standardSyncSummary=io.airbyte.config.StandardSyncSummary@75c2ea6f[status=failed,recordsSynced=0,bytesSynced=0,startTime=1681897354487,endTime=1681897356180,totalStats=io.airbyte.config.SyncStats@689c18c9[bytesCommitted=0,bytesEmitted=0,destinationStateMessagesEmitted=0,destinationWriteEndTime=1681897356179,destinationWriteStartTime=1681897354616,estimatedBytes=<null>,estimatedRecords=<null>,meanSecondsBeforeSourceStateMessageEmitted=0,maxSecondsBeforeSourceStateMessageEmitted=0,maxSecondsBetweenStateMessageEmittedandCommitted=0,meanSecondsBetweenStateMessageEmittedandCommitted=0,recordsEmitted=0,recordsCommitted=0,replicationEndTime=1681897356180,replicationStartTime=1681897354487,sourceReadEndTime=0,sourceReadStartTime=1681897354529,sourceStateMessagesEmitted=0,additionalProperties={}],streamStats=[],additionalProperties={}],normalizationSummary=<null>,webhookOperationSummary=<null>,state=<null>,outputCatalog=io.airbyte.protocol.models.ConfiguredAirbyteCatalog@4114fc87[streams=[io.airbyte.protocol.models.ConfiguredAirbyteStream@4e01e2ec[stream=io.airbyte.protocol.models.AirbyteStream@2cf2e8d1[name=my_stream,jsonSchema={"$schema":"http://json-schema.org/draft-07/schema","type":"object","required":["url"],"properties":{"url":{"description":"The URL for the API endpoint.","type":"string"},"token":{"description":"The API token for authentication.","type":"string"}}},supportedSyncModes=[full_refresh],sourceDefinedCursor=<null>,defaultCursorField=[],sourceDefinedPrimaryKey=[],namespace=<null>,additionalProperties={}],syncMode=full_refresh,cursorField=[],destinationSyncMode=overwrite,primaryKey=[],additionalProperties={}]],additionalProperties={}],failures=[io.airbyte.config.FailureReason@24a0e7d6[failureOrigin=source,failureType=system_error,internalMessage=401 Client Error:  for url: http://192.168.100.158:9098/category/build/1,externalMessage=Something went wrong in the connector. See the logs for more details.,metadata=io.airbyte.config.Metadata@4236d701[additionalProperties={attemptNumber=2, jobId=2, from_trace_message=true, connector_command=read}],stacktrace=Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 74, in handle_http_error
    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]
KeyError: 401

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/airbyte/integration_code/main.py", line 13, in <module>
    launch(source, sys.argv[1:])
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 131, in launch
    for message in source_entrypoint.run(parsed_args):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 122, in run
    for message in generator:
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py", line 110, in read
    stream_is_available, error = stream_instance.check_availability(logger, self)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py", line 190, in check_availability
    return self.availability_strategy.check_availability(self, logger, source)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 51, in check_availability
    return self.handle_http_error(stream, logger, source, error)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 81, in handle_http_error
    raise error
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 45, in check_availability
    get_first_record_for_slice(stream, stream_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py", line 38, in get_first_record_for_slice
    return next(records_for_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 423, in read_records
    yield from self._read_pages(
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 439, in _read_pages
    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 462, in _fetch_next_page
    response = self._send_request(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 364, in _send_request
    return backoff_handler(user_backoff_handler)(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 331, in _send
    raise exc
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 328, in _send
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
,retryable=<null>,timestamp=1681897355565,additionalProperties={}], io.airbyte.config.FailureReason@3614b29[failureOrigin=source,failureType=<null>,internalMessage=Source didn't exit properly - check the logs!,externalMessage=Something went wrong within the source connector,metadata=io.airbyte.config.Metadata@386e9ff4[additionalProperties={attemptNumber=2, jobId=2, connector_command=read}],stacktrace=io.airbyte.workers.internal.exception.SourceException: Source didn't exit properly - check the logs!
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:442)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1589)
Caused by: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.
	at io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150)
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:440)
	... 4 more
,retryable=<null>,timestamp=1681897355748,additionalProperties={}]],commitStateAsap=true,additionalProperties={}]
2023-04-19 09:42:36 [32mINFO[m i.a.c.i.LineGobbler(voidCall):149 - 
2023-04-19 09:42:36 [32mINFO[m i.a.w.t.s.ReplicationActivityImpl(lambda$replicate$3):202 - Sync summary length: 6950
2023-04-19 09:42:36 [32mINFO[m i.a.c.t.TemporalUtils(withBackgroundHeartbeat):307 - Stopping temporal heartbeating...
2023-04-19 09:42:36 [32mINFO[m i.a.p.j.e.LoggingJobErrorReportingClient(reportJobFailureReason):26 - Report Job Error -> workspaceId: 86069cca-1e19-40f1-b325-79feee3f1e24, dockerImage: airbyte/source-customize-api:0.0.1, failureReason: io.airbyte.config.FailureReason@35ebb160[failureOrigin=source,failureType=system_error,internalMessage=401 Client Error:  for url: http://192.168.100.158:9098/category/build/1,externalMessage=Something went wrong in the connector. See the logs for more details.,metadata=io.airbyte.config.Metadata@504e6d7[additionalProperties={attemptNumber=2, jobId=2, connector_command=read, from_trace_message=true}],stacktrace=Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 74, in handle_http_error
    reason = self.reasons_for_unavailable_status_codes(stream, logger, source, error)[status_code]
KeyError: 401

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/airbyte/integration_code/main.py", line 13, in <module>
    launch(source, sys.argv[1:])
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 131, in launch
    for message in source_entrypoint.run(parsed_args):
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py", line 122, in run
    for message in generator:
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py", line 110, in read
    stream_is_available, error = stream_instance.check_availability(logger, self)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/core.py", line 190, in check_availability
    return self.availability_strategy.check_availability(self, logger, source)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 51, in check_availability
    return self.handle_http_error(stream, logger, source, error)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 81, in handle_http_error
    raise error
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/availability_strategy.py", line 45, in check_availability
    get_first_record_for_slice(stream, stream_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/utils/stream_helper.py", line 38, in get_first_record_for_slice
    return next(records_for_slice)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 423, in read_records
    yield from self._read_pages(
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 439, in _read_pages
    request, response = self._fetch_next_page(stream_slice, stream_state, next_page_token)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 462, in _fetch_next_page
    response = self._send_request(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 364, in _send_request
    return backoff_handler(user_backoff_handler)(request, request_kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 331, in _send
    raise exc
  File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/streams/http/http.py", line 328, in _send
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 401 Client Error:  for url: http://192.168.100.158:9098/category/build/1
,retryable=<null>,timestamp=1681897355565,additionalProperties={}], metadata: {connector_repository=airbyte/source-customize-api, workspace_url=http://localhost:8000/workspaces/86069cca-1e19-40f1-b325-79feee3f1e24, failure_type=system_error, connector_command=read, connector_name=customize api 1, workspace_id=86069cca-1e19-40f1-b325-79feee3f1e24, airbyte_version=0.44.0, connection_url=http://localhost:8000/workspaces/86069cca-1e19-40f1-b325-79feee3f1e24/connections/20824119-a73c-46aa-988d-4dc49992134b, connector_definition_id=05985f94-09d0-4ee3-b5e4-c533d25a5f05, failure_origin=source, connection_id=20824119-a73c-46aa-988d-4dc49992134b, connector_release_stage=custom, job_id=2, deployment_mode=OSS}
2023-04-19 09:42:36 [32mINFO[m i.a.c.f.EnvVariableFeatureFlags(autoDisablesFailingConnections):36 - Auto Disable Failing Connections: false